applications/StructuralMechanicsApplication/custom_constitutive/truss_constitutive_law.h

```diff
@@ -123,7 +140,7 @@ class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION) TrussConstitutiveLaw : public
     ///@}
     ///@name Member Variables
     ///@{
-
+    double mStressState = 0.0; // The current stress state
```

Comment:
Do you need this even in the elastic case?

If you do then you should also serialize it